models/db/engine.go

```diff
@@ -216,17 +205,16 @@ func InitEngineWithMigration(ctx context.Context, migrateFunc func(*xorm.Engine)
 	return nil
 }
 
-// NamesToBean return a list of beans or an error
-func NamesToBean(names ...string) ([]interface{}, error) {
-	beans := []interface{}{}
+// TablesToBeans return a list of beans for tables or an error
+func TablesToBeans(names ...string) ([]interface{}, error) {
```

Comment:
The name here does not have to be a table name.

This function was designed back in #12407 to be quite a bit more forgiving and will take bean type names too.

For example, it would match `EmailHash`, `email_hash` and `schema.email_hash` with the `EmailHash` bean.

Reply:
Yes, so I didn't call it `TableNamesToBeans`

`Table` is only a concept to present  `EmailHash`, `email_hash` or `schema.email_hash`

Reply:
Another choice is `NamesToTableBeans`.

`db.NamesToBeans` is quite misleading. It seems can convert every names (variable/column, etc) in database to a bean